src/common/constants/quiz.ts

```diff
@@ -1,3 +1,40 @@
+  'Analyticals are task-oriented and unassertive. They are concerned with analytical processes and are persistent,systematic problem solvers. They can also be seen as aloof, picky, and critical. Analyticals are very securityconscious and have a high need to be right, leading them to an over reliance on data collection. In their questfor data, they tend to ask many questions about specifics. Their actions and decisions tend to be slow andextremely cautious, but they will rarely miss a deadline. Although they are great problem solvers, Analyticalscould be better decision-makers. Analyticals tend to be perfectionists, serious, and orderly. They focus on the details and the process of work,and become irritated by surprises and “glitches.” Their theme is, “Notice my efficiency” and their emphasis is oncompliance and working within existing guidelines to promote quality in products or service. Analyticals like organization and structure, and dislike too much involvement with other people. They workslowly and precisely by themselves, are time-disciplined, and prefer an intellectual work environment.Analyticals tend to be critical of their own performance. They tend to be skeptical and like to see things inwriting. Analyticals’ primary strengths are their accuracy, dependability, independence, follow-through, andorganization. Their primary weaknesses are their procrastination and conservative natures, which promote theirtendency to be picky and over-cautious. Occupations that they tend to gravitate toward are accounting,engineering, computer programming, the hard sciences (chemistry, physics, and math) systems analysis, andarchitecture.';
+
+const RELATIONSHIP_TEXT_ONE = '• Support their organized, thoughtful approach';
+const RELATIONSHIP_TEXT_TWO = '• Demonstrate through actions rather than words';
+const RELATIONSHIP_TEXT_THREE =
+  '• List advantages and disadvantages of any plan';
+const RELATIONSHIP_TEXT_FOUR = '• Provide solid, tangible, factual evidence';
+const RELATIONSHIP_TEXT_FIVE =
+  '• Provide guarantees that actions won’t backfire';
+
+const MANAGING_TEXT_ONE =
+  '• Motivate: Appeal to their need to be accurate and to their logical approach to things.';
+const MANAGING_TEXT_TWO =
+  '• Correct: Specify the exact behavior and outline how you would like to see the behavior changed. Establish checkpoints and timelines.';
+const MANAGING_TEXT_THREE =
+  '• Compliment: Praise their efficiency, though processes, organization, persistence, and accuracy.';
+const MANAGING_TEXT_FOUR =
+  '• Delegate: Take time to answer all their questions about structure and guidance. The more they understand the details, the more likely they will be to complete the task properly. Be sure to establish deadlines.';
+const MANAGING_TEXT_FIVE =
+  '• Counsel: Describe the process that you plan to follow. Outline how that process will produce the results they seek. Ask questions to help them give the right information. Let them show how much they know.';
```

Comment:
There is no need to store this in common constants, let's keep it as an object and move it to `quiz-rezults/common.ts`. But I would rather not store it separately from the component at all, because we are not going to change this text or use it somewhere else
```javascript
const quizText = {
  generalText: '',
  relationship: {
    text1: '',
    text2: '',
    ...
  },
  managing: {
    text1: '',
    text2: '',
    ...
  }
}
```